### DISCUSSION TITLE
Spreading the word

### DISCUSSION THREAD

**Discussion by JelleZijlstra** [python/typing#832] [General]
Sebastian, do you already have plans on how to get people to find this new forum? Some things we could do:
- Email typing-sig@ (and maybe other Python mailing lists)
- Announce in https://gitter.im/python/typing
- Put an issue in the trackers for the major type checkers
- Add a pointer to https://github.com/typeddjango/awesome-python-typing

**Comment by srittau** [comment]
All suggestions sound good, although I would also announce to python-list. Maybe also:

* Reach out to a blogger so that it gets announced on planet Python.
* Add a note to typeshed's README.

As a start, I'd like to "start small", until we have settled a bit on a structure and have first experience. For a start, I'd suggest:

* Announce on gitter.
* Add notes to typeshed and possible awesome-python-typing.

The next step would be to announce it on typing-sig (and possibly the type checker issue trackers, although I'd expect that the maintainer are also subscribed to typing-sig).

And then finally "public" announcements on typing-list and Planet Python.

**Reply by JelleZijlstra** [reply]
Sounds good! I just put up a note on gitter.

**Reply by srittau** [reply]
I opened python/typeshed#5901. I also opened #833 here to track this.

**Reply by sobolevn** [reply]
I've also shared the link in our `dry-python` community: https://t.me/drypython
We have quite a lot of typing related questions!

Great initiative 👍

**Reply by sobolevn** [reply]
And in https://gitter.im/mypy-django/Lobby

**Comment by abylinjohnson** [comment]
Use split() function...
